scenes/game_elements/characters/npcs/talker/components/default.dialogue

```diff
@@ -1,5 +1,18 @@
-# SPDX-FileCopyrightText: The Threadbare Authors
-# SPDX-License-Identifier: MPL-2.0
-~ start
-{{npc_name}}: [[Hi|Hello|Greetings]], {{player_name}}.
+
+~ start  
```

Comment:
Rather than modifying the placeholder dialogue, please make a new dialogue file in your storyquest directory.